client/lib/plans/features-list.js

```diff
@@ -42,7 +42,7 @@ import {
 	FEATURE_BLANK,
 	FEATURE_BLOG_DOMAIN,
 	FEATURE_BUSINESS_ONBOARDING,
-  FEATURE_CLOUDFLARE_ANALYTICS,
+	FEATURE_CLOUDFLARE_ANALYTICS,
```

Comment:
nit: please ensure these kinds of changes do not make it into the PR.

Reply:
@worldomonation - I did not make changes to these files deliberately, it could have happened when I was trying to set up test env. I was wondering if it would be ok to delete this file from the fork?